src/data.table.h

```diff
@@ -42,7 +42,11 @@
 /* we mean the encoding bits, not CE_NATIVE in a UTF-8 locale */
 #define IS_UTF8(x)  (getCharCE(x) == CE_UTF8)
 #define IS_LATIN(x) (getCharCE(x) == CE_LATIN1)
-#define IS_ASCII(x) (LEVELS(x) & 64) // API expected in R >= 4.5
+#if R_VERSION < R_Version(4, 5, 0) || R_SVN_REVISION < 86789
```

Comment:
Keeping this here even though there's another `(4,5,0)` check above at L21

 - The `R_SVN_REVISION`s are different
 -  The `IS_UTF8()` and `IS_LATIN()` macros are also right here